### ISSUE TITLE
[Installation] Files downloaded are not always master.zip

### ISSUE THREAD

**Issue by bylankai** [facebookresearch/LASER#45]
During installing the external library (bash install_external_tools.sh) I found that wget somehow the gave me 302 responses and redirected me to https://.../master so I ended up downloaded "master" instead of "master.zip". Therefore, the installation failed because the script was trying to find a file named "master.zip" which doesn't exist. So it's better to add an -O option for wget command I think.

**Comment by hoschwenk** [comment]
Thanks for pointing out this problem (it never happened during my tests).
When wget is redirected to another site, it still downloads the same version ?
(this is important for the Moses tokenizer since the latest version behaves slightly differently)

**Comment by bylankai** [comment]
Sorry for late reply.
I've binary checked the diff between the wget-downloaded file and the file I downloaded directly from fastBPE website and they're the same.
Actually I tried to download by some other machines and the filename never got changed so I guess the reason is that the proxy behind that particular server somehow removed the file extension. 

Therefore, I'm gonna close this since it's not really a general issue.
Thanks
